packages/kit-bg/src/services/ServiceMarketV2.ts

```
@@ -72,41 +87,285 @@ class ServiceMarketV2 extends ServiceBase {
+    timeTo,
+  }: {
+    tokenAddress: string;
+    networkId: string;
+    interval?: string;
+    timeFrom?: number;
+    timeTo?: number;
+  }) {
+    let innerInterval = interval?.toUpperCase();
+
+    if (innerInterval?.includes('M')) {
+      innerInterval = innerInterval?.toLowerCase()
```

ESLint

[error] Insert `;`

(prettier/prettier)